package NYT.NChunkClient.NProto;

import "yt_proto/yt/core/misc/proto/guid.proto";

////////////////////////////////////////////////////////////////////////////////

message TSessionId
{
    required NYT.NProto.TGuid chunk_id = 1;
    required int32 medium_index = 2;
}

////////////////////////////////////////////////////////////////////////////////
